pkgs/by-name/ag/age-plugin-tpm/package.nix

```
@@ -39,7 +41,7 @@ buildGoModule rec {
     mainProgram = "age-plugin-tpm";
     homepage = "https://github.com/Foxboron/age-plugin-tpm";
     license = licenses.mit;
-    platforms = platforms.linux;
+    platforms = platforms.all;
```

```suggestion
    platforms = platforms.unix;
```
more correct probably

---

If `platforms.unix` includes platforms without TPM support it's wrong.

---

`age` is not dependent on hardware support though. `go-tpm` (which `age-plugin-tpm` uses) support Linux and Windows, so encrypt+decrypt *probably* works? Apple platforms does not support TPMs at all.

Note I don't run Windows nor test things I write on there.

---

> Note I don't run Windows nor test things I write on there.

Ha I assumed, same here.

I will say the encrypt only use case works well for me on macOS. Thanks for fixing that.